src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig

```diff
@@ -44,14 +44,14 @@
 
     {% set icon %}
         <span class="sf-toolbar-label"><img src="{{ asset('themes/new-theme/img/glyph.png') }}" /></span>
-        <span class="sf-toolbar-value">{{ collector.applicationversion }}</span>
+        <span class="sf-toolbar-value">{{ constant('PrestaShop\\PrestaShop\\Core\\Version::VERSION') }}</span>
     {% endset %}
 
     {% set text %}
         <div class="sf-toolbar-info-group">
             <div class="sf-toolbar-info-piece">
-                <b>{{ collector.applicationname }}</b>
-                <span>{{ collector.applicationversion }}</span>
+                <b>PrestaShop</b>
```

Comment:
Not blocking but I think this should have been configure as global twig variable.

Reply:
Using a variable is interesting if it has dynamic value, you were thinking about the shop name maybe? Since it's only used in the debug toolbar I'd say PrestaShop hard-coded is not really a problem

Besides adding a global variable is only interesting if the variable is used in many places, which is unlikely in this case

Reply:
I disagree but it's not blocking for me :). Deal 👍